src/main/java/com/blackducksoftware/integration/jira/issue/handler/JiraIssueHandler.java

```diff
@@ -286,66 +287,79 @@ private void addComment(final BlackDuckIssueModel blackDuckIssueModel, final Str
+                String key = IssueTrackerHandler.createEntityPropertyKey(blackDuckIssueModel.getJiraIssueId());
                 issueServiceWrapper.getIssuePropertyWrapper().addProjectProperty(blackDuckIssueModel.getJiraIssueFieldTemplate().getProjectId(), jiraUserContext.getJiraAdminUser(), key, issueTrackerProperties);
-            } catch (final JiraIssueException e) {
+            } catch (JiraIssueException e) {
                 handleJiraIssueException(e, blackDuckIssueModel);
             }
         }
     }
 
-    private void updateDefaultIssueProperties(final BlackDuckIssueModel blackDuckIssueModel) {
-        final BlackDuckIssueFieldTemplate blackDuckIssueFieldTemplate = blackDuckIssueModel.getBlackDuckIssueFieldTemplate();
-        final IssueCategory category = blackDuckIssueFieldTemplate.getIssueCategory();
+    private void updateDefaultIssueProperties(BlackDuckIssueModel blackDuckIssueModel) {
+        BlackDuckIssueFieldTemplate blackDuckIssueFieldTemplate = blackDuckIssueModel.getBlackDuckIssueFieldTemplate();
+        IssueCategory category = blackDuckIssueFieldTemplate.getIssueCategory();
         if (!IssueCategory.SPECIAL.equals(category)) {
-            final IssueProperties properties = new IssueProperties(category, blackDuckIssueModel.getBomComponentUri(), blackDuckIssueFieldTemplate.getPolicyRuleName(), blackDuckIssueModel.getJiraIssueId());
+            IssueProperties properties = new IssueProperties(category, blackDuckIssueModel.getBomComponentUri(), blackDuckIssueFieldTemplate.getPolicyRuleName(), blackDuckIssueModel.getJiraIssueId());
+
+            String alertCategory;
+            if (category == IssueCategory.VULNERABILITY) {
+                alertCategory = "Vulnerability";
+            } else {
+                alertCategory = "Policy";
+            }
+
+            AlertIssueSearchProperties alertIssueSearchProperties = new AlertIssueSearchProperties("Black Duck", "Project", blackDuckIssueFieldTemplate.getProjectName(), "Project Version",
+                blackDuckIssueFieldTemplate.getProjectVersionName(), alertCategory,
+                "Component", blackDuckIssueFieldTemplate.getComponentName(), "Component Version", blackDuckIssueFieldTemplate.getComponentVersionName(), "");
+
             logger.debug("Setting default properties on issue: " + properties);
             try {
                 issueServiceWrapper.getIssuePropertyWrapper().addIssueProperties(blackDuckIssueModel.getJiraIssueId(), jiraUserContext.getJiraAdminUser(), blackDuckIssueModel.getBomComponentUri(), properties);
-            } catch (final JiraIssueException e) {
+                issueServiceWrapper.getIssuePropertyWrapper().addAlertIssueProperties(blackDuckIssueModel.getJiraIssueId(), jiraUserContext.getJiraAdminUser(), alertIssueSearchProperties);
+            } catch (JiraIssueException e) {
                 handleJiraIssueException(e, blackDuckIssueModel);
```

Comment:
These lines are the important changes to this file. The others should all be removing "final"